helm/portieris/templates/securitycontextconstraint.yaml

```diff
@@ -0,0 +1,44 @@
+{{ if .Capabilities.APIVersions.Has "clusterversions.config.openshift.io" }}
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: true
+allowPrivilegedContainer: false
+allowedCapabilities: []
+apiVersion: security.openshift.io/v1
+defaultAddCapabilities: []
+fsGroup:
+  type: RunAsAny
+groups:
+- system:cluster-admins
+kind: SecurityContextConstraints
```

Comment:
Just for readability, could `apiVersion`, `kind`, and `metadata` be moved to the top of this file please? It took me a while to realise this file had those fields. Alphabetical is fine for the other fields.